src/modules/logger/logger.h

```diff
@@ -301,13 +292,7 @@ class Logger : public ModuleBase<Logger>
 
 	void write_changed_parameters(LogType type);
 
-	inline bool copy_if_updated_multi(int sub_idx, int multi_instance, void *buffer, bool try_to_subscribe);
-
-	/**
-	 * Check if a topic instance exists and subscribe to it
-	 * @return true when topic exists and subscription successful
-	 */
-	bool try_to_subscribe_topic(LoggerSubscription &sub, int multi_instance);
+	inline bool copy_if_updated(int sub_idx, void *buffer, bool try_to_subscribe);
```

Comment:
Why `inline`? I think the compiler can figure that out given the optimization flags.

Reply:
No it does not. This makes a noticeable difference in CPU load.

Reply:
From an earlier collapsed comment - https://github.com/PX4/Firmware/pull/12123#discussion_r297249561

Reply:
Oh wow, ok!